lib/bolt/project.rb

```diff
@@ -47,9 +55,15 @@ def initialize(path, type = 'option')
       @resource_types = @path + '.resource_types'
       @type = type
 
-      @project_file = @path + 'bolt-project.yaml'
-      @data = Bolt::Util.read_optional_yaml_hash(File.expand_path(@project_file), 'project') || {}
-      validate if load_as_module?
+      yaml = Bolt::Util.read_optional_yaml_hash(File.expand_path(@project_file), 'project') || {}
```

Comment:
I thought we made sure `read_optional_yaml_hash` always returned a hash? Is this `nil` somehow? Maybe its a bug?

Reply:
I thought the exact same thing! Yes, there's a test that fails saying that it's nil. I wonder if the test mocks Bolt::Util and doesn't mock this function maybe?

Reply:
Yep - found it and fixed.

Reply:
What was the problem? I was thinking you wouldnt need the `|| {}`
```suggestion
yaml = Bolt::Util.read_optional_yaml_hash(File.expand_path(@project_file), 'project')
```